CMakeLists.txt

```diff
@@ -393,7 +393,12 @@ ELSE(CMAKE_TOOLCHAIN_FILE)
     MESSAGE(WARNING "No compiler identified")
   ENDIF()
   MESSAGE(STATUS "Compiler: ${COMPILER}")
-  MESSAGE(STATUS "Compiler Version: ${CMAKE_CXX_COMPILER_VERSION}")
+  # Workaround for not properly identified CXX compiler in case of IBM XL compiler
+  IF( ${COMPILER} MATCHES "IBM" )
+    MESSAGE(STATUS "Compiler Version: ${CMAKE_C_COMPILER_VERSION}")
+  ELSE()
+    MESSAGE(STATUS "Compiler Version: ${CMAKE_CXX_COMPILER_VERSION}")
+  ENDIF()
 
```

Comment:
Not always though - the first lines of output come from CMake's compiler tests which we have no influence on, while the later lines are our output. I agree that the version info may be redundant but I would urge to at least keep the compiler info as this is our only line of output to verify that our own tests chose wisely.